crates/core/src/serde_utils.rs

```diff
@@ -32,6 +32,25 @@ pub mod u256 {
     }
 }
 
+pub mod h160 {
```

Comment:
You added this mod here instead of inside `ef_tests` because we could need this functionality when we get transactions from the rpc endpoint?

Reply:
Not really, I just didn't know exactly where to add it. Should we move it to `ef_tests`?

Reply:
Yeah, I would probably move it to ef_tests then for now. We can add a new module serde_utils there

Reply:
Done!